src/mscorlib/shared/System/IO/Path.Windows.cs

```diff
@@ -165,20 +165,15 @@ public static string GetTempFileName()
             Span<char> initialBuffer = stackalloc char[PathInternal.MaxShortPath];
             var builder = new ValueStringBuilder(initialBuffer);
```

Comment:
It is not necessary to use `ValueStringBuilder` here anymore because of the buffer never grows.

Reply:
Ignore this comment. I have not notice that it is passed to Normalize.